panel/tests/ui/layout/test_feed.py

```diff
@@ -47,10 +48,10 @@ def test_feed_view_latest(page):
     # Assert scroll is not at 0 (view_latest)
     assert feed_el.evaluate('(el) => el.scrollTop') > 0
 
-    wait_until(lambda: int(page.locator('pre').last.inner_text()) > 950, page)
+    wait_until(lambda: int(page.locator('pre').last.inner_text()) > ITEMS * 0.95, page)
```

Comment:
```suggestion
    wait_until(lambda: int(page.locator('pre').last.inner_text()) > 0.9 * ITEMS, page)
```

Seems to be a bit less strict